CONTRIBUTING.md

```diff
@@ -90,32 +88,21 @@ where you feel less comfortable or need to learn something new.
 
 **tl;dr:**
 1. Work on a feature branch
-1. [Run linting and tests often](README.md#linting--running-tests)
-1. Ensure your change has test coverage (_ideally complete coverage_)
-1. Submit a descriptive pull request
+2. Submit a descriptive pull request
```

Comment:
Markdown has a neat feature where numbered lists can just be the number 1 for each line and it will render correctly, then you don't need to worry about later edits to the list making the ordering wonky. So this can just be `1. Submit a ...`